i18n/de/cosmic_term.ftl

```diff
@@ -49,6 +49,11 @@ default-dim-font-weight = Matte Schriftstärke
 default-bold-font-weight = Fette Schriftstärke
 use-bright-bold = Fetten Text heller darstellen
 
+### Behavior
+behavior = Behavior
```

Comment:
You don't have to fill in every translation file. It is best to only fill in the languages you know, and it always falls back to the English variant if a language misses a translation. This makes it easy for people translating other languages to see what is missing.

Reply:
At first I only added the English file but I got an error with the pl! macro

Reply:
Not sure if I'm doing it the wrong way, but it would be way easier to find what's missing if the changes were synced across files (since it often requires rearranging and careful checking, especially in large files).